### DISCUSSION TITLE
Shift off github

### DISCUSSION THREAD

**Discussion by broukema** [digidem/mapeo-mobile#797] [Ideas]
At https://www.digital-democracy.org/blog/localfirst/ there are a whole lot of very good arguments in favour of Mapeo. But then suddenly there's info that Mapeo is hosted on a [GAFAM](https://en.wikipedia.org/wiki/GAFAM) owned website - Github. Which is racist - blocking access to users in several countries.

It's self-defeating to support one of the world's worst Orwellian privacy violators - Microsoft (the M in GAFAM) - while trying to help develop autonomy and local data sovereignship and privacy.

You really should consider shifting off github and discouraging people from using it: https://archive.vn/U1R2L - it: favours propietary software; was bought by GAFAM; violates scientific ethics; ...  As I say there, I'm compromising too, but with the aim of helping people and communities to move from GAFAM to the free world.

One obvious option is https://codeberg.org , run by a citizens' association in Germany. (You could, for example, keep the github branch online, but block new issues and discussion and require issues and discussion to be posted on the new server instead.)

**Comment by okdistribute** [comment]
Thanks @broukema, I agree that Github puts users at risk and as it is a US company must comply with a ton of draconian rules and it sucks.

Have you tried https://gitea.io/en-us/?

**Comment by broukema** [comment]
hi Karissa @okdistribute . No I haven't tried gitea.io, but codeberg runs (I think) on a branch of gitea :). I don't see any specific list of sites *running* gitea and providing a service to users.

In [this thread on the Fediverse](https://mstdn.swiso.org/@switchingsoftware/104598028311938841), the [switching.software](https://switching.software) Fediverse contact personpointed me to:
* this nice short list: https://developers.reverseeagle.org/replace/github/ 
  * sourcehut/codeberg/notabug/disroot/gitea/gitgud/gitlab server list/gitea server list
* a wider discussion on whether or not to add lists of ethical tech services to switching.software: https://codeberg.org/swiso/website/issues/110

I've been running several git repositories on Codeberg for several months and I'm quite happy - I can't complain. The codeberg contact reacts quickly on the Fediverse when I think of something useful to say or ask - which is an important practical point in terms of shifting away from FB/Twitter.

There's [a project to add federated social networking](https://forgefed.peers.community/) (my [brief summary)](https://framapiaf.org/@boud/104481579650362891) across the ethical git repositories, to solve a major Tyranny of Convenience argument for staying on github.
